static/app/views/performance/transactionSummary/transactionTags/content.tsx

```diff
@@ -310,13 +346,25 @@ const StyledMain = styled('div')`
   max-width: 100%;
 `;
 
+const StyledSearchBar = styled(SearchBar)`
+  @media (min-width: ${p => p.theme.breakpoints.small}) {
+    order: 1;
+    grid-column: 1/6;
+  }
+
+  @media (min-width: ${p => p.theme.breakpoints.xlarge}) {
+    order: initial;
+    grid-column: auto;
+  }
```

Comment:
This is interesting vs. having the `FilterActions` `grid-template-columns` change on the breakpoint, is there a reason you went with changing the grid columns?

Reply:
I copied this from another `FilterActions` in the codebase, and it did the right thing so I didn't second guess it!

Reply:
Ah, no worries then 👍